docs/orchestration/rbac/overview.md

```diff
@@ -0,0 +1,88 @@
+# Role Based Access Controls <Badge text="Cloud"/>
+
+Prefect Cloud Enterprise plans come with customizable role based access controls.
+
+**Roles** are used to denote the permissions a user has as part of a team.
+
+Roles can be assigned to both users and service accounts.
+
+### Default Roles
+
+Prefect Cloud contains three default roles
+
+- Administrator
+- User
+- Restricted User
+
+Default roles cannot be modified.
+
+#### Administrators
+
+Administrators have all permissions given to users, but are also able to actively manage their team in Prefect Cloud. This means they are able to invite new members, remove unwanted members, and change the roles of other members of their team.
+
+#### Users
+
+Users have permission to perform all actions required for daily use of Prefect Cloud; this includes registering new flows, kicking off flow runs, etc. They do not have permission to perform actions that impact other members of the team, meaning they cannot change the roles of their teammates, invite others to their tenant, etc.
+
+#### Restricted Users
```

Comment:
These are currently called read only users in the UI. Let's coordinate to make sure we use the same terms in both places.

Reply:
good call, updated to "Read Only Users"